po/ko.po

```diff
@@ -8666,10 +8689,20 @@ msgid ""
 "  * Rust standard library is aware that types can implement `Default` and provides convenience "
```

Comment:
@mgeisler can you elaborate on this statement? I can't get what it means. What's "the convenience methods" the std lib provides?